test/distributed_tests_utils.jl

```diff
@@ -128,7 +152,7 @@ function run_tripolar_simulation(grid)
     set!(model, c = ηᵢ, η = ηᵢ)
 
     if architecture(grid) isa ReactantState || child_architecture(grid) isa ReactantState  
-        r_loop! = @compile sync=true raise=true loop!(model)
+        r_loop! = @compile sync=true raise=true reactant_loop!(model)
```

Comment:
Another test that would be sufficient is to compile a single time step and then loop over that, like

```julia
rstep! = @compile sync=true raise=true time_step!(model, dt)
for n = 1:100
    rstep!(model, dt)
end
```

Reply:
yeah, probably better